samples/console_output/main.cpp

```diff
@@ -1,38 +1,140 @@
 #include "sensr.h"
 #include <iostream>
+#include <string.h>
```

Comment:
Do we need any string operation on `char*`?

Reply:
yes, we use it to check arguments. e.g.) `if(strcmp(argv[i], "zone")==0)`

Reply:
I get it. FYI, there's an alternative named cstring :smile:

Reply:
Will change it to cstring.